astropy/io/fits/tests/test_core.py

```diff
@@ -23,7 +23,9 @@
 from astropy.utils import data
 from astropy.utils.compat.optional_deps import (
     HAS_BZ2,
-)  # NOTE: Python can be built without bz2.
+)
```

Comment:
OMG lol... the fix is wrong. @nstarman , how do I ask pre-commit to ignore this?

Reply:
nvm